evalml/data_checks/invalid_target_data_check.py

```diff
@@ -82,7 +82,7 @@ def validate(self, X, y):
 
             >>> X = pd.DataFrame({"col": [1, 2, 3, 1]})
             >>> y = pd.Series(["cat_1", "cat_2", "cat_1", "cat_2"])
-            >>> target_check = InvalidTargetDataCheck("regression", "R2")
+            >>> target_check = InvalidTargetDataCheck("regression", "R2", null_strategy="impute")
```

Comment:
it would be nice to keep the default behavior in the docstring. Can you file an issue for it?

Reply:
For sure!

Reply:
cheers!